checkers/stringXbytes_checker.go

```diff
@@ -0,0 +1,50 @@
+package checkers
+
+import (
+	"go/ast"
+	"strings"
+
+	"github.com/go-lintpack/lintpack"
+	"github.com/go-lintpack/lintpack/astwalk"
+	"github.com/go-toolsmith/astfmt"
+	"github.com/go-toolsmith/typep"
+)
+
+func init() {
+	var info lintpack.CheckerInfo
+	info.Name = "stringXbytes"
+	info.Tags = []string{"diagnostic", "experimental"}
+	info.Summary = "Detects redundant conversions between string and []byte"
+	info.Before = `copy(b, []byte(s))`
+	info.After = `copy(b, s)`
+
+	collection.AddChecker(&info, func(ctx *lintpack.CheckerContext) lintpack.FileWalker {
+		return astwalk.WalkerForExpr(&stringXbytes{ctx: ctx})
+	})
+}
+
+type stringXbytes struct {
+	astwalk.WalkHandler
+	ctx *lintpack.CheckerContext
+}
+
+func (c *stringXbytes) VisitExpr(expr ast.Expr) {
+	if x, ok := expr.(*ast.CallExpr); ok {
+		switch name := qualifiedName(x.Fun); name {
```

Comment:
Could you rewrite 1-case switch to an if statement?
Then the CI will pass and we can merge it.

Reply:
fixed